src/test/python/jointGenerator/rockInput.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/python
+import numpy as np
+import sys
+from Joint import Joint
+import math
+
+# Reads input text file containing data on rock volume and joints
```

Comment:
Would you mind adding some more detailed documentation? I think we should specify the precise structure of the input and output files here.